.github/workflows/compliance.yml

```diff
@@ -5,6 +5,7 @@ jobs:
       - uses: mtfoley/pr-compliance-action@main
         with:
           body-auto-close: false
+          body-regex: PLEASE_NEVER_MATCH_I_DONT_USE_ISSUES
```

Comment:
Oh, does the action not provide a straightforward way to disable this?

Reply:
Not that I could find, either in the README or in the code. But I'd love to hear about it if I missed it.